brigadier/brigadier/settings.py

```diff
@@ -150,7 +150,6 @@
 ]
 
 if DEBUG and pkgutil.find_loader('debug_toolbar'):
-    import debug_toolbar
```

Comment:
why do you deleted debug_toolbar ?

Reply:
because it's not necessary import. Toolbar works without it.